ports/winpcap/portfile.cmake

```
@@ -178,21 +161,17 @@ if(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
         set(PCAP_BINARY_PATH "${PCAP_BINARY_PATH}/x64")
     endif()
 
-    file(
-        INSTALL
+    file(INSTALL
             "${PCAP_BINARY_PATH}/Packet.dll"
             "${PCAP_BINARY_PATH}/wpcap.dll"
-        DESTINATION
-            ${CURRENT_PACKAGES_DIR}/bin
-    )
+        DESTINATION ${CURRENT_PACKAGES_DIR}/bin)
 
-    file(
-        INSTALL
+    file(INSTALL
             "${PCAP_BINARY_PATH}/Packet.dll"
             "${PCAP_BINARY_PATH}/wpcap.dll"
-        DESTINATION
-            ${CURRENT_PACKAGES_DIR}/debug/bin
-    )
+        DESTINATION  ${CURRENT_PACKAGES_DIR}/debug/bin)
 endif()
 
-configure_file(${COPYRIGHT} ${CURRENT_PACKAGES_DIR}/share/winpcap/copyright COPYONLY)
+vcpkg_replace_string(${CURRENT_PACKAGES_DIR}/include/pcap-stdinc.h "#define inline __inline" "/* #define inline __inline */")
```

This macro may still be required in C, in which case it should be
```suggestion
vcpkg_replace_string(${CURRENT_PACKAGES_DIR}/include/pcap-stdinc.h "#define inline __inline" "#ifndef __cplusplus\n#define inline __inline\n#endif")
```

---

Updated, thanks!

---

Updated, thanks!